### PR TITLE
Improve polaris delete

### LINKED ISSUE #1034: Adding -y support to delete pvc in synopsysctl delete polaris command
**What would you like to be added**:
Adding -y support to delete pvc in synopsysctl delete polaris command

**Why is this needed**:
Some may automate to create and delete polaris command, for them, it will be easy to add -y to delete the pvc and underlying pv.

### PR DESCRIPTION
**Pull request by jdartigalongue**
Closes #1034
CLOUD-491

### CODE REVIEW

**Comment by operator-r2d2** [comment]
@jdartigalongue: once the present PR merges, I will cherry-pick it on top of master in a new PR and assign it to you.

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1047#issuecomment-547937917):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

File: pkg/polaris/ctl_polaris.go [hunk 1]
```diff
@@ -126,8 +126,8 @@ func (ctl *CRSpecBuilderFromCobraFlags) AddCRSpecFlagsToCommand(cmd *cobra.Comma
 	cmd.Flags().StringVar(&ctl.Registry, "registry", ctl.Registry, "Docker registry e.g. docker.io/myuser")
 
 	// Those flags need be hidden since external databases are not currently supported
```
Comment (yashbhutwala) [inline pkg/polaris/ctl_polaris.go]: Can you change this comment to make sure we're consistent? 😄

File: pkg/synopsysctl/cmd_delete.go [hunk 2]
```diff
@@ -200,14 +216,18 @@ var deletePolarisCmd = &cobra.Command{
...
-				log.Warn("Do you want to to delete the following PVCs?")
-				for _, v := range list.Items {
-					log.Warn(v.Name)
-				}
+				deletePvc := promptAnswerYes
 
-				deletePvc, err := askYesNo()
-				if err != nil {
-					return err
+				if !promptAnswerYes {
```
Comment (yashbhutwala) [inline pkg/synopsysctl/cmd_delete.go]: Is it possible to write a little unit test here?

File: pkg/synopsysctl/cmd_delete_test.go [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+			args: args{
+				question: func() error {
+					return nil
+				},
+				answer:    "yEs",
+				alwaysYes: false,
+			},
+			want:    true,
+			wantErr: false,
+		},
```
Comment (mphammer) [inline pkg/synopsysctl/cmd_delete_test.go]: I think we should throw in a case for "" and "1#@7" so that we can test out `wantErr: true`

**Comment by operator-r2d2** [comment]
[APPROVALNOTIFIER] This PR is **APPROVED**

This pull-request has been approved by: *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1047#" title="Author self-approved">jdartigalongue</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1047#pullrequestreview-309264513" title="Approved">mphammer</a>*, *<a href="https://github.com/blackducksoftware/synopsys-operator/pull/1047#pullrequestreview-310106696" title="LGTM">yashbhutwala</a>*

The full list of commands accepted by this bot can be found [here](https://go.k8s.io/bot-commands?repo=blackducksoftware%2Fsynopsys-operator).

The pull request process is described [here](https://git.k8s.io/community/contributors/guide/owners.md#the-code-review-process)

<details >
Needs approval from an approver in each of these files:

- ~~[OWNERS](https://github.com/blackducksoftware/synopsys-operator/blob/release-2019.11.x/OWNERS)~~ [jdartigalongue,mphammer,yashbhutwala]

Approvers can indicate their approval by writing `/approve` in a comment
Approvers can cancel approval by writing `/approve cancel` in a comment
</details>

**Comment by operator-r2d2** [comment]
@jdartigalongue: new pull request created: #1050

<details>

In response to [this](https://github.com/blackducksoftware/synopsys-operator/pull/1047#issuecomment-547937917):

>/cherrypick master


Instructions for interacting with me using PR comments are available [here](https://git.k8s.io/community/contributors/guide/pull-requests.md).  If you have questions or suggestions related to my behavior, please file an issue against the [kubernetes/test-infra](https://github.com/kubernetes/test-infra/issues/new?title=Prow%20issue:) repository.
</details>

**Pull request merged**